<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFormatFile</class>
 <widget class="QDialog" name="AddFormatFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление формата файла - RayTracing</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Наименование:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/complete-button.png</normaloff>images/complete-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/cancel-button.png</normaloff>images/cancel-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_used">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Используется:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="used">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>62</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
